src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -187,9 +188,9 @@ $facility-locator-color-gray: #ccc;
   }
 
   .facility-search-results {
-    ol {
-      padding: 0;
-    }
+    margin: 0 0 0 0em ;
+    padding-left: 0em;
+    list-style-type: none;
```

Comment:
Having list-style-type: none here will remove the semantic context of the list. 

* [accessible list guidance](https://www.scottohara.me/blog/2018/05/26/aria-lists.html)

* ["Fixing" Lists](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)

Due to the design, we may have to use `div role="list"` instead of an `ol", and then `div role="listitem"` for each of the `li` items.